prov/efa/src/efa_hmem.c

```diff
@@ -85,26 +126,48 @@ static int efa_hmem_support_status_update_cuda(struct efa_hmem_support_status *c
 		return ret;
 	}
 
-	cuda_status->p2p_supported = true;
+	cuda_info->p2p_supported = true;
```

Comment:
There need to be some comment around line 117.

On a platform that does not support p2p, we exited there, so none of the variables were set.

Meanwhile, on such system, we do not expect to get cuda pointer (for now).

So some comment is needed saying that in the future, when we support cuda on non-gdr platform, we need to set those variables for non-p2p cases.

Reply:
done